lib/cousins.ml

```diff
@@ -6,13 +6,10 @@ open Util
 
 let default_max_cnt = 2000
 
-let max_cousin_level conf base p =
+let max_cousin_level conf _base _p =
```

Comment:
if they _base and _p are unused maybe remove them

Reply:
ok